src/applications/personalization/profile/components/personal-information/ContactInformationEditView.jsx

```diff
@@ -80,6 +80,29 @@ export class ContactInformationEditView extends Component {
     }
   }
 
+  focusOnUpdateButton() {
+    const focusableElement = this.editForm?.querySelector(
```

Comment:
I think all this code could be replaced with `focusElement('[data-testid="save-edit-button"]')` after you `import { focusElement } from 'platform/utilities/ui';`

Reply:
I tried this I thought. I will try it out again